index.bs

```diff
@@ -1696,6 +1696,7 @@ associated.
         a user's avatar or a [=[RP]=]'s logo. This URL MUST be an [=a priori authenticated URL=]. [=Authenticators=] MUST
         accept and store a 128-byte minimum length for an icon member's value.
         Authenticators MAY ignore an icon member's value if its length is greater than 128 bytes.
+        The URL's scheme MAY be "data" to avoid fetches of the URL, at the cost of needing more storage.
```

Comment:
s/ scheme / [=scheme=] /

..which ought to link to https://url.spec.whatwg.org/#concept-url-scheme

if not can do [=concept-url-scheme|scheme=]  (IIUC)